charts/openshift-metering/templates/hive/hive-scripts-configmap.yaml

```diff
@@ -81,6 +81,21 @@ data:
     ln -s -f /hive-config/hive-log4j2.properties $HIVE_HOME/conf/hive-log4j2.properties
     ln -s -f /hive-config/hive-exec-log4j2.properties $HIVE_HOME/conf/hive-exec-log4j2.properties
 
+    # for databse configuration to mysql
+    # if a databse secret exists, check hive-site.xml, to ensure no values added yet. Inject with username and password stored
+    # in volume as database secret
+    if [ -f /var/run/secrets/metastore-db-secrets ]; then
```

Comment:
Just noticed a couple more things:
1. This is the wrong directory name it's `metastore-db-secret` and not `metastore-db-secrets`
2. We should be checking if the directory exists using the `-d` flag, instead of checking if that file exists

Reply:
That works